userController.js

```diff
@@ -0,0 +1,26 @@
+import { user } from '../models';
+ import { downloadResource } from '../util';
+
+ const controller = {};
+
+ controller.download = async (req, res) => {
+  const fields = [
+    {
+      label: 'Student ID',
+      value: 'googleID'
+    },
+    {
+      label: 'Student Name',
+      value: 'name'
+    },
+    {
+     label: 'Attendance Percentage',
+      value: 'attendance_percentage'
+    }
+  ];
+  const data = await user.findAll();
```

Comment:
User data does not contain the attendance_percentage directly, You will need to calculate it.